parser-library/parse.cpp

```diff
@@ -358,7 +362,8 @@ bool parse_resource_table(bounded_buffer *sectionData,
         return false;
       }
 
-      resource rsrc = {};
```

Comment:
This is not enough to set all fields to zero (GCC complained about uninitialized members), so I have added a memset.